includes/classes/SupportMonitor/Monitor.php

```diff
@@ -686,4 +686,17 @@ public function get_users_report() {
 
 		return $report;
 	}
+
+	/**
+	 * Check if the site is using an external object cache.
+	 *
+	 * @return bool
+	 */
+	public function get_is_using_object_cache() {
+		if ( wp_using_ext_object_cache() ) {
+			return true;
+		}
+
+		return file_exists( WP_CONTENT_DIR . '/object-cache.php' );
```

Comment:
@christianc1 This may not be a part of what you were working on, but I was under the impression we wanted to try to try to set the object cache and verify that a test variable is able to be cached to catch situations like this where hosting companies have a different setup: https://gitlab.10up.com/10up-internal/system-report/system-report-dashboard/-/issues/7

Reply:
That's actually not a viable method unfortunately.  I did try it though, see 690ddfc, specifically what was removed in that changeset.  The problem with that approach is that, if there is no `object-cache.php` drop-in, WordPress will load the core `WP_Object_Cache` class.  It's non-persistent, but is present for the entire request, so will pass the simple test:

```php
wp_cache_set( 'foo', 'bar' );

return ( 'bar' === wp_cache_get( 'foo' ) );
```

In the issue you reference, the plugin that was giving us a false positive uses the drop-in, so this method should detect it's presence: https://objectcache.pro/docs/deployment/